contacts-app/src/components/Input/Input.module.css

```diff
@@ -9,3 +9,8 @@
   border-radius: 4px;
   width: 60%;
 }
+.invalidMessage {
+  color: rgb(247, 121, 99);
+  font-size: 85%;
```

Comment:
Weird to see font sizes as a percent, though I'm not sure if its a good or a bad thing haha! perhaps paging @informa?

Reply:
Good question, I just did it as a quick fix. 
I think I read somewhere, I should convert px to rem in a global style, and use rem for font-size for individual module?
Is this the standard approach? @informa?